reference/5.0/Microsoft.PowerShell.Core/About/about_Requires.md

```diff
@@ -49,13 +53,30 @@ Example:
   #Requires -Modules Hyper-V
   ```
 
-You might think that the above code should not run because the required module
+You might think that the above code shouldn't run because the required module
 was removed before the `#Requires` statement. However, the `#Requires` state
 had to be met before the script could even execute. Then the first line of the
 script invalidated the required state.
 
 ### Parameters
 
+#### -Assembly \<Assembly path> | \<.NET assembly specification>
+
+Specifies the path to a `.dll` file or a .NET assembly specification. The
+**Assembly** parameter was introduced in PowerShell 5.0. For more information
+about .NET assemblies, see [Assemblies in .NET](/dotnet/standard/assembly).
+
```

Comment:
Make this change in all files.

```markdown
#### -Assembly \<Assembly path> | \<.NET assembly name>

Specifies the path to the assembly DLL file or a .NET assembly name. The 
**Assembly** parameter was introduced in PowerShell 5.0. For more information
about .NET assemblies, see [Assembly names](/dotnet/standard/assembly/names).
```

Reply:
Fixed **Assembly** parameter's description in v5.0 -7.